### ISSUE TITLE
Move our DockerHub account to something more consistent with the GitHub organization name

### ISSUE THREAD

**Issue by mumoshu** [actions-runner-controller/actions-runner-controller#566]
Extracted from https://github.com/actions-runner-controller/actions-runner-controller/issues/562#issuecomment-846336222

DockerHub doesn't allow hyphens in the organization name so it's impossible to just use `actions-runner-controller`.

I've taken [actionsrunnercontroller](https://hub.docker.com/orgs/actionsrunnercontroller) anyway but not yet sure if this is good enough.

**Comment by sollie** [comment]
Have you considered hosting packages on GitHub/ghcr.io instead or in addition to DockerHub?

**Comment by mumoshu** [comment]
Nope. Does it help? Anyone's using it as a primary container image source?

**Comment by mabushey** [comment]
Docker the company is really imploding. They have been placing so many bandwidth restraints on DockerHub that it's getting unusable. I recommend ditching docker/dockerhub. There are so many container engines that docker itself is irrelevant. Even Kubernetes is deprecating Docker as a container runtime after v1.20.: https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/

**Comment by sledigabel** [comment]
Dockerhub is sure losing some momentum.
Github container registry will integrate better with github actions too, the permissions are attached to the repository and it will integrate better with github actions as you don't need to add a username/password

https://github.com/marketplace/actions/build-and-publish-docker-image-to-github-packages-registry

**Comment by mumoshu** [comment]
Thanks everyone. I hear you all. But this is about using another DockerHub account and all you're talking about how ghcr.io is great! I appreciate the information, but I don't see what you're asking.

Perhaps stop publishing to DockerHub while adding ghcr.io as the single registry that we publish images? But then I can easily imagine that another person doesn't like ghcr.io will start asking me to restart publishing to DockerHub. So it's very unlikely we stop using DockerHub.

To be fair, K8s deprecating Docker as a container runtime has nothing to do with DockerHub. I usually use dockerd or contained as my k8s container runtimes, and gcr.io, qyay.io, and docekrhub in my work. I do prefer using contained these days, but still using DockerHub. I don't dislike quay.io even though my experience says it's still more unreliable than DockerHub :)

Anyway, I'm just explaining my thoughts. If you want us to add ghcr.io as the another container registry to publish images, just create another issue, or even better contribute a pull request to enhance our github actions workflow to support it.
